src/clj/rems/config.clj

```diff
@@ -52,6 +52,9 @@
   (when-let [url (:public-url config)]
     (assert (.endsWith url "/")
             (str ":public-url should end with /:" (pr-str url))))
+  (assert (some (partial = (:default-language config)) (:languages config))
```

Comment:
I like the idiom `#{(:default-language config)}` for these.

Reply:
I actually had that at first, but since the expression gets printed in the exception, I switched to the "clearer" `partial =`. Now I'm not so sure...